include/RAJA/Timer.hxx

```diff
@@ -91,7 +91,7 @@ public:
                  + std::chrono::microseconds(tstart.tv_usec);
     auto stop = std::chrono::seconds(tstop.tv_sec)
                 + std::chrono::microseconds(tstop.tv_usec);
-    telapsed += std::chrono::duration<double>(stop - start).count();
+    telapsed += std::chrono::duration<long double>(stop - start).count();
```

Comment:
Hmmm, maybe we pull Duration out of the class definition (or put it in TimerBase), should work okay. Might be worth checking though, I don't know if this could be necessary BG/Q oddness.